### ISSUE TITLE
[Bug]: Registry remapping faliure?

### ISSUE THREAD

**Issue by ssmorkss** [Virtuoel/Pehkui#367]
### Minecraft version(s)

1.19.2

### Minecraft version details

1.19.2

### Mod loader

Fabric

### Mod loader version

1.19.2

### Fabric API version

1.19.2

### In what kind of world or server did the problem occur?

My multiplayer server that I was running

### What went wrong? (Crash logs don't go here)

Everything was fine at first. I kept downloading the wrong version of the source code so I fixed it and got the right version, but now when I try to join the server it won't let me in and keeps giving me this dumb code I don't understand. 

![image_2023-01-24_115753171](https://user-images.githubusercontent.com/123437068/214357612-be536f1c-769c-4a97-be4d-1d7cb0c19257.png)

### Full list of installed mods and their version numbers

![image_2023-01-24_115737569](https://user-images.githubusercontent.com/123437068/214357543-9b3a4a20-cac0-41cf-9e02-4dd8eb4035be.png)

### Did the problem cause the game to crash?

Yes. (paste crash report below)

**Comment by Virtuoel** [comment]
You still have the wrong versions in that screenshot (3.6.3 Forge, and 3.5.0 Fabric).
Remove both of those and try using the [3.6.3 Fabric jar](https://modrinth.com/mod/pehkui/version/3.6.3+1.14.4-1.19.3).

**Comment by ssmorkss** [comment]
So wait, delete both of them and add that? Or what

> You still have the wrong versions in that screenshot (3.6.3 Forge, and 3.5.0 Fabric). Remove both of those and try using the [3.6.3 Fabric jar](https://modrinth.com/mod/pehkui/version/3.6.3+1.14.4-1.19.3).

**Comment by Virtuoel** [comment]
Correct.

**Comment by ssmorkss** [comment]
> When I do that it gives me an error.

![image_2023-01-24_135419610](https://user-images.githubusercontent.com/123437068/214382859-5c356305-2e4b-4b6d-ac91-32fef440bc5e.png)

**Comment by Virtuoel** [comment]
You need to update Fabric Loader by rerunning the Fabric installer

**Comment by ssmorkss** [comment]
> You need to update Fabric Loader by rerunning the Fabric installer

Ohhhh! Okay that makes a lot more sense, thanks! Doing it now, what loader vers should I do? :) I have options 0.14.13 and downwards.

**Comment by Virtuoel** [comment]
Loader works on every MC version, so never hurts to update. Grab the latest version, so 0.14.13.

**Comment by ssmorkss** [comment]
> Loader works on every MC version, so never hurts to update. Grab the latest version, so 0.14.13.

I'm not going to lie, I'm trying everything. nothing is working & I'm not sure what I'm doing wrong. The server is the right version, I'm using the right mod & loader. But despite that it still isn't working.

**Comment by Virtuoel** [comment]
Hmm, I'm also unsure. Can you post screenshots of both your client's `mods` folder and the server's `mods` folder?

**Comment by ssmorkss** [comment]
> Hmm, I'm also unsure. Can you post screenshots of both your client's `mods` folder and the server's `mods` folder?

Yooo! okay so I got it figured out. Apparently I was doing everything perfectly fine! The only issue was that my server wasn't working and I couldn't have been able to fix it unless I contacted the hosting site. They fixed the Fabric version for me! All I need now is the Pehkui Mod for 1.19.2, which one(s) should I download? Thank you so much for your help, and sorry for being such a pain in the arse.
